packages/reports/src/Stores/Modules/Profit/Helpers/__tests__/format-request.spec.ts

```diff
@@ -0,0 +1,40 @@
+import getDateBoundaries from '../format-request';
+
+describe('getDateBoundaries', () => {
+    it('should return date_from and date_to', () => {
+        const date_from = 1635724800;
+        const date_to = 1635811200;
+        const partial_fetch_time = false;
+        const should_load_partially = false;
+        const returnValue = getDateBoundaries(date_from, date_to, partial_fetch_time, should_load_partially);
+        expect(returnValue).toEqual({
+            date_from: 1635724800,
```

Comment:
If `1635724800` is equal to constant value is 5 line, then you can just omit it, like:
`.toEqual({date_from, date_to})`

Reply:
Same in some other places, but it's optional, so ignore if irrelevant